### DISCUSSION TITLE
Welcome to fontawesome Discussions!

### DISCUSSION THREAD

**Discussion by rich-iannone** [rstudio/fontawesome#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by joekeane7** [comment]
Hi Rich. 

I am a keen user of Font Awesome in R studio and recently downloaded the latest version (0.3.0) hopeing it would have all the icons available in the Font Awesome webiste: https://fontawesome.com/search?q=time&s=solid%2Cbrands

Is there a way to get missing icons attached to the FontAwesome Library in R Studio?

Many thanks,

Joe.

**Reply by sanjmeh** [reply]
I am also a keen user but moving from free icons to pro icons the process looks daunting. Still searching an easy way.
